### PR TITLE
Added a program to calculate trailing zero

### PR DESCRIPTION
**Pull request by rsshalini**

### CODE REVIEW

File: .vscode/.ropeproject/config.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# The default ``config.py``
+
+
+def set_prefs(prefs):
```
Comment (orsenthil) [inline .vscode/.ropeproject/config.py:4]: Why did you check this in `.ropeproject/config.py ` ?

File: 06_30_2018/trailingzero.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!usr/bin/python
+
+#counts the number of trailing zero for n factorial
+
+import math
+def count_trailing_zero_n_fact(n):
+    n_factorial = str(math.factorial(n))
+    trailing_zero_count = 0
+    if n_factorial.endswith('0'):
+        for i in reversed(n_factorial):
```
Comment (orsenthil) [inline 06_30_2018/trailingzero.py]: Calling `reversed(n_factorial)` on every iteration of the for loop can be confusing. You could do something like this.

```
reversed_factorial = reversed(n_factorial)
for digit in reversed_factorial:
   # do more
```

**Review by orsenthil — CHANGES_REQUESTED** [review]

File: .vscode/.ropeproject/config.py (hunk 1 above)
Comment (rsshalini) [reply .vscode/.ropeproject/config.py:4]: Not sure how it came. removed it now

File: .vscode/.ropeproject/objectdb [hunk 3]
```diff
@@ -0,0 +1 @@
+�}q.
```
Comment (orsenthil) [inline .vscode/.ropeproject/objectdb]: You added another file.

**Pull request merged**